runtime/doc/lua.txt

```diff
@@ -2926,6 +2926,8 @@ vim.fs.normalize({path}, {opts})                          *vim.fs.normalize()*
       • {opts}  (`table?`) Options:
                 • expand_env: boolean Expand environment variables (default:
                   true)
+                • unique_slash: boolean Remove duplicated slashes (default:
```

Comment:
The parameter name `unique_slash` sounds a bit awkward. I can suggest:
- `normalize_slash` (or `normalize_slashes`)
- `collapse_slash` (or `collapse_slashes`)

Normalize slash would be a more common term used in other programming languages' stdlib.

Reply:
Nice one. I like `collapse_slash`.